File: python/program/Course.py

class Course:
    
    def __init__(self, name, code):
        self._name = name
        self._code = code
    
    def setName(self, name):
        self._name = name
    
    def setCode(self, code):
        self._code = code
    
    def getName(self):
        return self._name 

    def getCode(self):
        return self._code

File: java/program/Course.java
public class Course {
    private String name;
    private String code;

    public Course(String name, String code){
        this.name = name;
        this.code = code;
    }

    void setName(String name){
        this.name = name;
    }

    void setCode(String code){
        this.code = code;
    }

    String getName(){
        return this.name;
    }

    String getCode(){
        return this.code;
    }
}
